assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -10,32 +10,44 @@ As an administrator, you can disable the telemetry data collection feature based
 
 {company-name} collects and analyzes the following data:
 
+*Web Analytics (Segment)*
+
+Web Analytics tracks user behavior and interactions with {product}. Specifically, it tracks the following:
+
 * Events of page visits and clicks on links or buttons.
-* System-related information, for example, locale, timezone, user agent including browser and OS details.
+* System-related information, for example, locale, time zone, user agent including browser and operating system details.
 * Page-related information, for example, title, category, extension name, URL, path, referrer, and search parameters.
 * Anonymized IP addresses, recorded as `0.0.0.0`.
 * Anonymized username hashes, which are unique identifiers used solely to identify the number of unique users of the {product-very-short} application.
 
-With {product-very-short}, you can customize the telemetry data collection feature and the telemetry Segment source configuration based on your needs.
+*System Observability (OpenTelemetry)*
+
+System Observability tracks the performance of the {product-very-short}. Specifically, it tracks the following metrics:
+
+* Key system metrics such as CPU usage, memory usage, and other performance indicators.
+* Information about system components, such as the locale, time zone, and user agent (including details of the browser and operating system).
+* Traces and logs monitor system processes, allowing you to troubleshoot potential issues impacting the performance of {product-very-short}.
+
+With {product-very-short}, you can customize the _Web Analytics_ and _System Observability_ configuration based on your needs.
 
 
 // disabling telemetry
-include::modules/observe/ref-disabling-telemetry.adoc[leveloffset=+1]
+include::modules/observe/analytics/ref-disabling-telemetry.adoc[leveloffset=+1]
```

Comment:
For consistency, and to avoid unintended side effects, consider avoiding multiplying the subdirectory nesting in the modules directory: create the directory directly in the `modules` directory.

For consistency, consider using the same name for the title (`telemetry`) and the modules subdirectories.

```suggestion
include::modules/telemetry/ref-disabling-telemetry.adoc[leveloffset=+1]
```